README.md

```diff
@@ -88,6 +89,22 @@ data:
   AWS_SECRET_ACCESS_KEY: $IDP_AWS_SECRET_ACCESS_KEY_BASE64
 EOF
 kubectl apply -f ./aws-secrets-eobs.yaml
+
+# AWS Credentials for flux-system Namespace for TOFU Controller
+cat << EOF > ./aws-secrets-tofu.yaml
+---
+apiVersion: v1
+kind: Secret
+metadata:
+name: aws-credentials
+namespace: flux-system
+type: Opaque
+stringData:
+AWS_ACCESS_KEY_ID: ${YOUR_AWS_ACCESS_KEY_ID}
+AWS_SECRET_ACCESS_KEY: ${YOUR_AWS_SECRET_ACCESS_KEY}
+AWS_REGION: ${IDP_AWS_REGION}
```

Comment:
the format fo rthis secret is not right

Reply:
Seriously idk how this got messed up in doc. Fixed it.